.github/workflows/release-drafter-pr.yml

```diff
@@ -12,5 +9,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: release-drafter/release-drafter@v5
+        with:
+          disable-releaser: true
```

Comment:
Nice! How did you find it? It's not documented 😞

Reply:
I know, I had to go read the code! https://github.com/release-drafter/release-drafter/blob/master/action.yml#L46